examples/example.xbb

```diff
@@ -16,14 +16,31 @@
 # Blackbird quantum assembly language example
 # ===========================================
 # Blackbird is case sensitive.
-# Newlines at the same indentation level indicate
-# the end of a statement.
+# Newlines indicate the end of a statement.
 # Like Python, we use # for comments.
 
+# -------------------------------------------
+# Metadata
+# -------------------------------------------
+# At the top of the file, we define:
+#
+# * name: the name of the quantum program,
+# * version: the Blackbird version the script targets
+# * target: (optional) device/simulator target
+#   that the quantum program has been compiled for.
+#
+# In addition, keyword options can be provided for the
+# target using the syntax (option1=0.34, option2=True)
```

Comment:
```suggestion
# target using the syntax `(option1=0.34, option2=True)`.
```
Not sure if this is better, but I think we want to highlight which part is code

Reply:
Sure. Using `(option1=0.34, option2=True)` is a stopgap solution, since kwargs were already supported by the grammar.

If you prefer something else, I can do that in another PR. Another option I considered:

```python
target chip0
    - shots=100
```

But this is a minor change, so I decided it was out of scope of this PR for now.

Reply:
No, I just meant that in the comment, we should put quotes to emphasize which parts are code vs comments